ftw/simplelayout/handlers.py

```diff
@@ -14,11 +14,11 @@ def unwrap_persistence(conf):
     """Unwrap recursice persistent page state
     """
     def unwrap(data):
-        if isinstance(data, PersistentMapping):
+        if isinstance(data, PersistentMapping) or isinstance(data, dict):
             data = dict(data)
             for key, value in data.items():
                 data[key] = unwrap(value)
-        elif isinstance(data, PersistentList):
+        elif isinstance(data, PersistentList) or isinstance(data, list):
```

Comment:
Please also include `tuple` and `set` here
- we always want to recurse (it might contain a dict, right?)
- we want to convert tuples and sets to lists too, since json knows only lists
```python
if isinstance(data, (PersistentList, list, tuple, set)):
```